webpack/scenes/ContentViews/Details/Versions/ContentViewVersionErrata.js

```diff
@@ -15,6 +15,10 @@ const ContentViewVersionErrata = ({ cvId, versionId, errataCounts }) => {
     total,
   } = errataCounts;
 
+  if (!Number(total)) {
+    return <InactiveText text={__('No applicable errata')} />;
+  }
+
```

Comment:
!Number(0) returns true..Didn't know that..

Reply:
derp